core/network/fabric-interop-cc/interfaces/asset-mgmt/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.16
 
 require (
 	github.com/golang/protobuf v1.5.2
-	github.com/hyperledger-labs/weaver-dlt-interoperability/common/protos-go v1.2.3
+	github.com/hyperledger-labs/weaver-dlt-interoperability/common/protos-go v1.3.0
```

Comment:
I don't see a similar version upgrade in the `go.mod` files in `libs/assetexchange` and `libs/utils`. Can you check if all submodules are covered?

Reply:
Done.